src/parameters/Microphysics1M.jl

```diff
@@ -153,16 +153,17 @@ struct CloudIce{FT, PD, MS} <: CloudCondensateType{FT}
     r_ice_snow::FT
     "deposition sublimation non_equil microphysics relaxation timescale [s]"
     τ_relax::FT
-    "ice density [kg/m3]"
-    ρi::FT
+    "cloud ice apparent density [kg/m3]"
+    ρᵢ::FT
 end
 
 CloudIce(::Type{FT}) where {FT <: AbstractFloat} =
     CloudIce(CP.create_toml_dict(FT))
 
+#:cloud_ice_apparent_density => :ρi,
```

Comment:
Are these comments leftover?

Reply:
I'll change them in the next PR. I just left them there to remember to what density from `ClimaParams.jl` I want to actually switch to